src/main/java/hybrid/it/internship/library/entity/Book.java

```diff
@@ -0,0 +1,39 @@
+package hybrid.it.internship.library.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "books")
+public class Book {
+
+    @Getter @Setter
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "book_id", unique = true, nullable = false)
+    private long id;
+
+    @Getter @Setter
+    @Column(name = "author", unique = false, nullable = false)
```

Comment:
No need for @Column annotation if field is named the same as in DB (field_name = fieldName).
@NotNull annotation can be used instead of nullable=false.
Unique should be kept on database level.